components/Landing/LandingCardItemsFirst.tsx

```diff
@@ -0,0 +1,33 @@
+import Image from 'next/image';
+
+import IconBox from './Iconbox';
+
+const LandingCardItemsFirst = () => {
+  return (
+    <div className="relative h-pr-419 w-pr-996 rounded-pr-40 bg-gradient-to-r from-brand-primary to-brand-tertiary p-pr-1 mo:h-pr-467 mo:w-pr-343 ta:h-pr-354 ta:w-pr-696">
+      <div className="relative flex size-full rounded-pr-40 bg-b-primary shadow-lg backdrop-blur-md mo:flex-col-reverse">
+        <div className="flex h-full w-1/2 items-end justify-end mo:absolute mo:bottom-0 mo:w-full mo:items-end mo:justify-center">
+          <Image
+            src="/images/landing/img-LandingCardimgfirst.png"
+            alt="LandingCardimgfirst"
+            width={291}
+            height={338}
+            className="mr-pr-33 h-pr-338 w-pr-291 object-contain mo:m-0 mo:h-pr-273 mo:w-pr-235 ta:mr-pr-7 tamo:h-pr-273 tamo:w-pr-235"
+          />
+        </div>
+
+        <div className="flex h-full w-1/2 items-center justify-start mo:w-full mo:items-start mo:justify-start">
+          <div className="bottom-pr-124 ml-pr-160 flex h-pr-120 flex-col gap-pr-16 text-24m mo:absolute mo:left-pr-54 mo:top-pr-48 mo:m-0 mo:h-pr-106 mo:w-pr-118 ta:ml-pr-110 tamo:h-pr-106 tamo:w-pr-118 tamo:text-18m">
```

Comment:
여기도 tamo: 를 쓰셔서 태블릿이랑 모바일 둘다 width랑 height를 적용해주셨는데 모바일 속성이 중복으로 들어가있는것같아요
아래와 같이 수정 해주시고 여기 말고도 15번줄에도 중복 코드가 있네요 해당 중복부분 전체적으로 확인하셔서 수정해주세요!
```suggestion
          <div className="bottom-pr-124 ml-pr-160 flex h-pr-120 flex-col gap-pr-16 text-24m mo:absolute mo:left-pr-54 mo:top-pr-48 mo:m-0 ta:ml-pr-110 tamo:h-pr-106 tamo:w-pr-118 tamo:text-18m">
```

Reply:
@junghwa1996 page 파일 및 랜딩 컴포넌트 코드 확인 후 단위 누락된 코드까지 작업했습니다 작업했습니다 감사합니다.